classes/PodsInit.php

```diff
@@ -487,10 +487,10 @@ public function register_assets() {
 		 */
 		do_action( 'pods_before_enqueue_scripts', $suffix_min );
 
-		wp_register_script( 'pods-json', PODS_URL . 'ui/js/jquery.json.js', array( 'jquery' ), '2.3' );
+		wp_register_script( 'pods-json', PODS_URL . 'ui/js/jquery.json.js', array( 'jquery' ), '2.3', true );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: d5ae3a69e1ad1dfc1a7df08185d7347d</sub>